tutorials/containers/using-gitlab-as-a-container-registry/using-gitlab-as-a-container-registry.md

```diff
@@ -0,0 +1,244 @@
+---
+id: using-gitlab-as-a-container-registry
+summary: Learn how to use GitLab as a private container registry
+categories: containers
+tags: kubernetes, containers, gitlab, cicd
+difficulty: 3
+status: draft
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+published: 2019-09-02
+author: Peter De Sousa <peter.de.sousa@canonical.com>
+---
+
+# Using GitLab as a container registry
+
+## Introduction
+
+The aim of this tutorial post is to demonstrate how you can use Charmed Kubernetes in a CI/CD environment using GitLab’s container registry functionality.
+
+GitLab has the ability to store upto 10GB in a container registry for projects, you can incorporate the building of these containers into your own CI/CD pipeline or you can use Gitlab’s own CI/CD functionality to do this for you. For the purposes of this blog post you will do this by hand so you can get a grasp of the process.
+
+## Before you start
+
+Find yourself a healthy kubernetes cluster, or if you don’t have access to a cluster, head over to [MicroK8s](https://tutorials.ubuntu.com/tutorial/install-a-local-kubernetes-with-microk8s#0) and install one on your laptop at no cost to yourself, if you’re on Windows you can follow [this guide](https://itnext.io/microk8s-on-windows-the-canonical-way-ed15fd4e5476).
+
+![Charmed Kubernetes](./images/charmed_kubernetes.png)
+
+## Process
+To create our container registry on gitlab you will need to complete the following steps:
+
+- Create a project
```

Comment:
Could you please add at least a couple of words about each of those bullet points? This page looks a little empty at the moment.